modules/serverless-create-domain-mapping.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+//  * serverless/networking/serverless-domain-mapping.adoc
+
+[id="serverless-create-domain-mapping_{context}"]
+= Creating custom domain mapping for a service
+
+To map a custom domain name to a service, you must create a `DomainMapping` custom resource (CR).
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have created a Knative service, and own a custom domain that you want to map to that service.
++
+[NOTE]
+====
+Your custom domain must point to the DNS of the Knative cluster. Instructions for completing this step will vary based on your domain registrar.
+====
+
+.Procedure
+
+. Create a YAML file containing the `DomainMapping` CR in the same namespace as the Knative service you want to map to:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1alpha1
+kind: DomainMapping
+metadata:
+ name: <domain_name> <1>
+ namespace: default <2>
+spec:
+ ref:
+   name: <service_name> <3>
+   kind: Service
+   apiVersion: serving.knative.dev/v1
+----
+<1> The custom domain name that you want to map to the service.
+<2> The namespace of both the `DomainMapping` CR and the Knative service.
+<3> The name of the service to map to the custom domain.
+
+. Apply the `DomainMapping` CR as a YAML file:
```

Comment:
It feels superfluous, but I've always included a description of what `<filename>` is, like:

> Replace `<filename>` with the name of the YAML file.

Reply:
I believe they can call the YAML whatever they want because users can make a bunch of different domain mappings for services, so it might be too specific / more confusing to give a specific file name here.

Reply:
It seems a bit much IMO to do this every time we use `<filename>`, I think it's self explanatory from the step itself tbh that it should be their YAML file's name.